Portfolio/index.html

```diff
@@ -6,8 +6,98 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  <body style="margin: 0; padding: 0px">
+    <h2>Juneja.</h2>
+<ul>
+  <li>About</li>
+  <li>Skills</li>
+  <li>Project</li>
+  <li>Reviews</li>
+</ul>
+<button class="contact" >Contact me</button>
+    <h2>Vanshika Juneja </h2>
+    <img class="projectimg" src="/public/johnDoe.png">
+    <h3>GET  TO KNOW ME</h3>
+    <p> I am an ambitious and driven engineering student in my final year. I have a strong skill set in HTML, CSS, Javascript, Typescript, React, Sql and Python, and I have my sights set on front-end development and design as a profession. I am also a professional dancer and have strong leadership and management skills. I currently hold a 9.6 CGPA from Medicaps University and am the Operation Head at GDSC. An impressive academic and creative individual, I am sure to make a mark in the tech industry soon.<br><br>
+      <ul>
+     <li> Name: Vanshika Juneja</li>
+     <li> Age: 20</li>
+     <li> Education: Graduating Student</li>
+     <li> Currently in final year.</li>
+     <li> Computer-Science Engineer</li>
+      </ul>
+    </p>
+    <p>
+      <h3>SKILLS</h3>
+        <p><ul>
+          <li><b>Technical Skills:</b></li>
+  <br>
+  <ol>
+          <li>Front end: html css js, ts, jsx , react</li>
+          <li>Core Subjects : oop, cn , dbms, os</li>
+          <li>Others: git, sql, netlify, ms office, canva, figma </li>
+          <li>Tools: VS code, github , slack , discord, replit, notion</li>
+          <li>Programming Languages: C++, Python</li>
+  </ol><br>
+  <li><b> Soft Skiils: </b> </li>
+  <br>
+  <ol>
+  <li>Writing : Content writing, technical writing.
+  </li>
+  <li>Presentation skill</li>
+  <li> Public Speaking</li>
```

Comment:
There should not be a paragraph tag surrounding the 'PROJECTS' heading. Remove the opening <p> tag at line 46 and the closing </p> tag at line 79.